### PR TITLE
Added optional params for regexInclude and regexExclude

### PR DESCRIPTION
**Pull request by shubik**
By adding optional params regexInclude and / or regexExclude certain files can be skipped, r.g. fonts or SVGs.

### CODE REVIEW

**Comment by shubik** [comment]
Eric, hi, it would be great if you could approve my pull request. Look here: https://github.com/ehynds/grunt-image-embed/issues/33#issuecomment-66451774 -- people have same issue.

**Comment by shubik** [comment]
Thanks!

File: tasks/lib/encode.js [hunk 1]
```diff
@@ -105,11 +112,6 @@ exports.init = function(grunt) {
             }
           }
 
-          // Test for scheme less URLs => "//example.com/image.png"
-          if (!is_local_file && rSchemeless.test(loc)) {
-            loc = 'http:' + loc;
-          }
```
Comment (ehynds) [inline tasks/lib/encode.js:111]: Why was this logic removed?

Comment (shubik) [inline tasks/lib/encode.js:111]: It's strange that it was removed. If it was done by me, than it's an error.

Comment (ehynds) [inline tasks/lib/encode.js:111]: no worries. fixed in e60b7b9740b18c8a63b7d8fea4d719eb7be56368 & 4468ebbe2a6510a2ed86a30fc10c95770df8fa98

**Pull request merged**